tests/standAloneClient/metadataObjectChanged.xml

```diff
@@ -0,0 +1,143 @@
+				<eventObjectType>metadataEventNotifications.Metadata</eventObjectType> <!-- EventNotificationEventObjectType::METADATA -->
+				<eventConditions objectType="array">
+					<item objectType="KalturaFieldMatchCondition">
+						<description>Object type is entry</description>
+						<field objectType="KalturaEvalStringField">
+							<code>$scope->getEvent()->getObject()->getObjectType()</code>
+						</field>
+						<values objectType="array">
+							<item objectType="KalturaStringValue">
+								<value>1</value>
+							</item>
+						</values>
+					</item>
+				</eventConditions>
+				<format>1</format>
+				<subject>Custom Data Changed: {entry_name}/ ID: {entry_id} </subject>
+				<body><![CDATA[ Hello,<p>An entry's custom data object has changed.</p><p>Entry Name: {entry_name} <br>Entry ID:  {entry_id}<br>Entry Creator: {creator_name}, {creator_id}/{creator_email} </p>]]></body>
+				<fromEmail>{from_email}</fromEmail>
+				<fromName>{from_name}</fromName>
+				<to objectType="KalturaEmailNotificationStaticRecipientProvider">
+					<emailRecipients objectType="array">			
+						<item objectType = "KalturaEmailNotificationRecipient">
+							<email objectType="KalturaStringValue">
+								<value>{owner_email}</value>
+							</email>
+							<name objectType="KalturaStringValue">
+								<value>{owner_name}</value>
+							</name>
+						</item>
+					</emailRecipients>
+				</to>
+				<cc objectType="KalturaEmailNotificationStaticRecipientProvider">
+					<emailRecipients objectType="array">			
+						<item objectType = "KalturaEmailNotificationRecipient">
+							<email objectType="KalturaStringValue">
+								<value>{creator_email}</value>
+							</email>
+							<name objectType="KalturaStringValue">
+								<value>{creator_name}</value>
+							</name>
+						</item>
+					</emailRecipients>
+				</cc>
+				<contentParameters objectType="array">
+					<item objectType="KalturaEventNotificationParameter">
+						<key>from_email</key>
+						<description>Sender email</description>
+						<value objectType="KalturaEvalStringField">
+							<code>kConf::get("partner_notification_email")</code>
+						</value>
+					</item>
+					<item objectType="KalturaEventNotificationParameter">
+						<key>from_name</key>
+						<description>Sender name</description>
+						<value objectType="KalturaEvalStringField">
+							<code>kConf::get("partner_notification_name")</code>
+						</value>
+					</item>
+					<item objectType="KalturaEventNotificationParameter">
+						<key>creator_name</key>
+						<description>Entry creator name</description>
+						<value objectType="KalturaEvalStringField">
+							<code>!is_null(entryPeer::retrieveByPk($scope->getEvent()->getObject()->getObjectId())) ? entryPeer::retrieveByPk($scope->getEvent()->getObject()->getObjectId())->getkuser()->getFirstName() . ' ' . entryPeer::retrieveByPk($scope->getEvent()->getObject()->getObjectId())->getkuser()->getLastName() : ''</code>
```

Comment:
Please use $scope->getObject, no need for the getEvent in the middle, this comment relevant to all the conditions and parameters.